site/pages/our-work/case-study/fortnums/index.js

```diff
@@ -0,0 +1,280 @@
+            <source srcSet={headerMedium} media="(min-width: 690px)" />
+            <source srcSet={headerLarge} media="(min-width: 980px)" />
+            <img src={headerSmall} alt="Fortnum & Mason basket contents" className={styles.header__image} />
+          </picture>
+        </div>
+      </div>
+    </div>
+    <div className={styles.content}>
+      <div className={styles.content__wrapper}>
+        <h1 className={styles.content__mainTitle}>
+          Fortnum & Mason’s new, elegant website increases revenue and conversion rates
+        </h1>
+        <h2 className={styles.content__title}>
+          <span className={styles.content__redTitle}>
+            {"Let's make things better."}
+          </span>
+          Scaling the site, decreasing drop-outs and bringing the in-store experience online
+        </h2>
+        <p className={styles.content__paragraph}>
+          Fortnum & Mason knew its digital offer didn’t match the premium experience or
+          service being executed by its stores. With more and more shoppers heading online
+          to make purchases from over 130 different countries, it realised something
+          needed to change quickly.
+        </p>
+        <img src={tabletImage} className={cx('imageLeft', 'floatLeft')} alt="Tablet" />
+        <p className={styles.content__paragraph}>
+          The in-store experience at Fortnum’s embodies elegance and creativity. It
+          displays its renowned, high quality products in beautifully curated layouts and
+          delivers consistent, amazing customer service.  In total contrast, the old
+          website had an inconsistent design, was inflexible, and difficult to update. The
+          old technology it used meant that revenue was being lost due to high levels of
+          customer drop-out at certain stages in the buying process. For example, the
+          presentation of delivery options was confusing and if one element of a
+          customer’s order was unavailable, it could result in the entire order not going
+          out for delivery. A scalable, highly flexible new build was needed, one that
+          could replicate the elegance of Fortnum’s in-store experience and handle complex
+          orders and global deliveries.
+        </p>
+      </div>
+      <img src={allScreensImage} className={styles.allScreensImage} alt="All screens" />
+      <div className={styles.content__wrapper}>
+        <h2 className={styles.content__title}>
+          <span className={styles.content__redTitle}>
+            Do the right thing. Do the thing right.
+          </span>
+          Embracing organisational change and implementing cutting-edge technology
+        </h2>
+        <div className={cx('contactBox', 'showMedium')}>
+          <h2 className={styles.contactBox__heading}>
+            Project in mind?
+          </h2>
+          <a href="#contactUs" className={styles.contactBox__button}>Tell us more</a>
```

Comment:
We put a workaround into `routes/index.js` to handle #contactUs in the Home page Url (we had to jump to contact us across different routes and the hash doesn't work well with client side navigation). This workaround is messing up your contact us link. To see why, try this
* Go to your page
* Click the Tell us more
* Press back

Result: the Url still has contact us hash in - but it shouldn't.

You can fix this by adding a check for the `home` key in `routes\defintions`
```
    if (hrefElement.hash === '#contactUs') {
      const { state, data } = stateNavigator.parseLink(url);
      if (state.key === 'home') {
        url = stateNavigator.getNavigationLink(state.key, { ...data, contactUs: true });
      }
    }
```